--- a/configure.ac
+++ b/configure.ac
@@ -2513,7 +2513,6 @@ ext/timidity/Makefile
 ext/webp/Makefile
 ext/xvid/Makefile
 ext/zbar/Makefile
-po/Makefile.in
 docs/Makefile
 docs/plugins/Makefile
 docs/libs/Makefile
--- a/Makefile.am
+++ b/Makefile.am
@@ -2,7 +2,7 @@ DISTCHECK_CONFIGURE_FLAGS=--enable-gtk-d
 
 SUBDIRS = \
 	gst-libs gst sys ext pkgconfig \
-	m4 common docs tests po tools
+	m4 common docs tests tools
 
 DIST_SUBDIRS = \
 	gst gst-libs sys ext pkgconfig \
