--- a/Makefile
+++ b/Makefile
@@ -113,8 +113,8 @@ libpiano.so.0: ${LIBPIANO_RELOBJ} ${LIBP
 			-o libpiano.so.0.0.0 ${LIBPIANO_RELOBJ} \
 			${LIBWAITRESS_RELOBJ} ${LIBGNUTLS_LDFLAGS} ${LIBGCRYPT_LDFLAGS} \
 			${LIBJSONC_LDFLAGS}
-	ln -s libpiano.so.0.0.0 libpiano.so.0
-	ln -s libpiano.so.0 libpiano.so
+	ln -sf libpiano.so.0.0.0 libpiano.so.0
+	ln -sf libpiano.so.0 libpiano.so
 	${AR} rcs libpiano.a ${LIBPIANO_OBJ} ${LIBWAITRESS_OBJ}
 
 
@@ -199,8 +199,8 @@
 install-libpiano:
 	install -d ${DESTDIR}/${LIBDIR}/
 	install -m644 libpiano.so.0.0.0 ${DESTDIR}/${LIBDIR}/
-	ln -s libpiano.so.0.0.0 ${DESTDIR}/${LIBDIR}/libpiano.so.0
-	ln -s libpiano.so.0 ${DESTDIR}/${LIBDIR}/libpiano.so
+	ln -sf libpiano.so.0.0.0 ${DESTDIR}/${LIBDIR}/libpiano.so.0
+	ln -sf libpiano.so.0 ${DESTDIR}/${LIBDIR}/libpiano.so
 	install -m644 libpiano.a ${DESTDIR}/${LIBDIR}/
 	install -d ${DESTDIR}/${INCDIR}/
 	install -m644 src/libpiano/piano.h ${DESTDIR}/${INCDIR}/
