--- a/Makefile.am
+++ b/Makefile.am
@@ -7,7 +7,7 @@ aclocaldir = $(datadir)/aclocal
 aclocal_DATA = gst-element-check-@GST_API_VERSION@.m4
 
 SUBDIRS = pkgconfig \
-	gst libs plugins tests \
+	gst libs plugins \
 	po \
 	m4 \
 	common
@@ -18,7 +18,7 @@ endif
 
 # These are all the possible subdirs
 DIST_SUBDIRS = pkgconfig \
-	gst libs plugins tools tests \
+	gst libs plugins tools \
 	po \
 	m4 \
 	common
