--- a/configure.ac
+++ b/configure.ac
@@ -769,7 +769,6 @@ libs/gst/helpers/Makefile
 libs/gst/net/Makefile
 plugins/Makefile
 plugins/elements/Makefile
-po/Makefile.in
 tests/Makefile
 tests/benchmarks/Makefile
 tests/check/Makefile
--- a/Makefile.am
+++ b/Makefile.am
@@ -8,7 +8,6 @@ aclocal_DATA = gst-element-check-@GST_AP
 
 SUBDIRS = pkgconfig \
 	gst libs plugins \
-	po \
 	m4 \
 	common
 
@@ -19,7 +18,6 @@ endif
 # These are all the possible subdirs
 DIST_SUBDIRS = pkgconfig \
 	gst libs plugins tools \
-	po \
 	m4 \
 	common
 
